setup/initialize_database.sh

```diff
@@ -1,9 +1,36 @@
 #!/bin/sh
-echo "Setting up mysql database."
+echo "Setting up mysql database..."
 sudo mysql -uroot < acm.sql
-echo "Setting up submissions and problems data."
-mkdir ../auacm/app/data
-cp data.zip ../auacm/app/data/data.zip
-cd ../auacm/app/data
-unzip data.zip
-rm data.zip
+echo "Setting up test database..."
+echo "DROP DATABASE IF EXISTS acm_test; CREATE DATABASE acm_test; USE acm_test;\n" | mysql -uroot < acm.sql
+
+# Ask to purge the data directory if it exists
+if [ -e ../auacm/app/data ]
+then
+  echo "Do you want to reset the data directory? (y/n)"
+  read purge
+fi
+if [ "$purge"="y" ]
+then
+  echo "Purging and resetting data directory..."
+  rm -rf ../auacm/app/data
+  rmdir ../auacm/app/data
```

Comment:
I think this is unnecessary. I get an error from this line when I run the script.